### PR TITLE
support minimum image crop size

### PR DESCRIPTION
**Pull request by aliillyas**
- support minimum crop size

### CODE REVIEW

**Review by andreikushner — APPROVED** [review]

File: lib/src/androidTest/java/com/soundcloud/android/crop/CropBuilderTest.java [hunk 1]
```diff
@@ -63,6 +63,15 @@ public void testMaxSizeSetAsExtras() {
         assertThat(intent.getIntExtra("max_y", 0)).isEqualTo(300);
     }
 
+    public void testMinSizeSetAsExtras() {
+        builder.withMaxSize(200, 200);
```
Comment (ifeomaro) [inline lib/src/androidTest/java/com/soundcloud/android/crop/CropBuilderTest.java]: Shouldn't this be `builder.withMinSize(200, 200)`?

**Review by ifeomaro — COMMENTED** [review]
I added a comment, other than that, this looks good.

File: lib/src/androidTest/java/com/soundcloud/android/crop/CropBuilderTest.java (hunk 1 above)
Comment (aliillyas) [reply lib/src/androidTest/java/com/soundcloud/android/crop/CropBuilderTest.java]: good catch let me update that

**Comment by aliillyas** [comment]
@iRuth fixed the test

**Review by ifeomaro — APPROVED** [review]

**Pull request merged**